### PR TITLE
fio: add page

### PR DESCRIPTION
**Pull request by Dzordzu**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known): 3.27

### CODE REVIEW

File: pages/common/fio.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# fio
+
+> Flexible I/O tester
```
Comment (git-em) [inline pages/common/fio.md:3]:
```suggestion
> Flexible I/O tester.
```

File: pages/common/fio.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+> Tool that will spawn a number of threads or processes doing a particular type of I/O action.
+> More examples: <https://docs.oracle.com/en-us/iaas/Content/Block/References/samplefiocommandslinux.htm>.
+> More information: <https://fio.readthedocs.io/en/latest/fio_doc.html>.
+
+- Test random reads:
+
+`sudo fio --filename=device name --direct=1 --rw=randread --bs=4k --ioengine=libaio --iodepth=256 --runtime=120 --numjobs=4 --time_based --group_reporting --name=iops-test-job --eta-newline=1 --readonly`
```
Comment (navarroaxel) [inline pages/common/fio.md:10]: We use `path/to/file` and `snake_case` for placeholders.
```suggestion
`sudo fio --filename={{path/to/file}} --direct=1 --rw=randread --bs=4k --ioengine=libaio --iodepth=256 --runtime=120 --numjobs=4 --time_based --group_reporting --name={{job_name}} --eta-newline=1 --readonly`
```

File: pages/common/fio.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+- Test sequential reads:
+
+`sudo fio --filename=device name --direct=1 --rw=read --bs=4k --ioengine=libaio --iodepth=256 --runtime=120 --numjobs=4 --time_based --group_reporting --name=iops-test-job --eta-newline=1 --readonly`
+
+- Test random read/write:
+
+`sudo fio --filename=/custom mount point/file --size=500GB --direct=1 --rw=randrw --bs=4k --ioengine=libaio --iodepth=256 --runtime=120 --numjobs=4 --time_based --group_reporting --name=iops-test-job --eta-newline=1`
```
Comment (navarroaxel) [inline pages/common/fio.md:18]:
```suggestion
`sudo fio --filename={{path/to/file}} --size=500GB --direct=1 --rw=randrw --bs=4k --ioengine=libaio --iodepth=256 --runtime=120 --numjobs=4 --time_based --group_reporting --name={{job_name}} --eta-newline=1`
```

File: pages/common/fio.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+> More examples: <https://docs.oracle.com/en-us/iaas/Content/Block/References/samplefiocommandslinux.htm>.
+> More information: <https://fio.readthedocs.io/en/latest/fio_doc.html>.
+
+- Test random reads:
+
+`sudo fio --filename=device name --direct=1 --rw=randread --bs=4k --ioengine=libaio --iodepth=256 --runtime=120 --numjobs=4 --time_based --group_reporting --name=iops-test-job --eta-newline=1 --readonly`
+
+- Test sequential reads:
+
+`sudo fio --filename=device name --direct=1 --rw=read --bs=4k --ioengine=libaio --iodepth=256 --runtime=120 --numjobs=4 --time_based --group_reporting --name=iops-test-job --eta-newline=1 --readonly`
```
Comment (navarroaxel) [inline pages/common/fio.md:14]:
```suggestion
`sudo fio --filename={{path/to/file}} --direct=1 --rw=read --bs=4k --ioengine=libaio --iodepth=256 --runtime=120 --numjobs=4 --time_based --group_reporting --name={{job_name}} --eta-newline=1 --readonly`
```

File: pages/common/fio.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# fio
+
+> Flexible I/O tester
+> Tool that will spawn a number of threads or processes doing a particular type of I/O action.
+> More examples: <https://docs.oracle.com/en-us/iaas/Content/Block/References/samplefiocommandslinux.htm>.
```
Comment (navarroaxel) [inline pages/common/fio.md:5]: We only use 1 link per page
```suggestion
```

**Comment by git-em** [comment]
Perhaps `--number-of-jobs` could be omitted, since it's optional, potentially increasing compatibility with older devices (if i understood it well -- i've never used `fio`).

File: pages/common/fio.md (hunk 5 above)
Comment (Dzordzu) [reply pages/common/fio.md:5]: It's a pity that link to the best examples has to be removed :/

**Comment by Dzordzu** [comment]
> Perhaps `--number-of-jobs` could be omitted, since it's optional, potentially increasing compatibility with older devices (if i understood it well -- i've never used `fio`).

In my opinion `--numjobs` is quite useful, especially when used within single node testing. I would prefer to keep it

**Review by CleanMachine1 — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**